### ISSUE TITLE
paq-2-piq as a loss function returns two values

### ISSUE THREAD

**Issue by shshojaei** [chaofengc/IQA-PyTorch#55]
Hi :) it's a nice library! i have a question about paq-2-piq IQA as a Loss function, it returns 2 values like:
`tensor([[59.6956], [73.4137]])`
i think it should returns just one value , i.e. one score.
what are these two values?

**Comment by chaofengc** [comment]
The network returns scores for every image in the batch, it seems that you have two images as inputs.

Besides, **please make sure that you know details of the specific metric when you want to use it as loss function**. Some of them are not fully differentiable.
